app/controllers/code_schools_controller.rb

```diff
@@ -7,4 +7,16 @@ def index
     @code_schools_by_state = code_schools.by_state
     @country = Country.named('United States')
   end
+
+  def map
+    code_schools = CodeSchools.new.all
+    @lat_longs = []
+
+    code_schools.each do |hash| 
```

Comment:
Hey @rickr when you mean `move this to the CodeSchools` I'm sorry, but what are you referring too?